src/common/HaskellDo/Toolbar/State.hs

```diff
@@ -134,7 +137,26 @@ update ToggleError state = do
     localIO toggleError
     return state
 
+update ConvertToPDF state = do
+    checkIfInstalled <- atRemote . localIO $ readProcess "find" ["/usr/bin","-name","wkhtmltopdf"] []
+    (errorCode,_,_) <- atRemote . localIO $ readCreateProcessWithExitCode (shell "which wkhtmltopdf") ""
```

Comment:
Hi @javiertoledo! Thats true, I think with the _which_ command would be enough, my mistake 👍